scaffolder-templates/gitlab-workflows/advanced-workflow/template.yaml

```diff
@@ -0,0 +1,435 @@
+      if: ${{ parameters.quayConfig == 'Create New Repository' }}
+      input:
+        baseUrl: ${{ parameters.baseUrl }}
+        token: ${{ parameters.token }}
+        name: ${{ parameters.quayRepoName }}
+        visibility: ${{ parameters.visibility }}
+        description: ${{ parameters.newRepoDescription }}
+        repoKind: ${{ parameters.repoKind }}
+        namespace: ${{ parameters.quayNamespace }}
+    - id: workflowCodeTemplate
+      name: Generating the Workflow Source Code and Catalog Info Component
+      action: fetch:template
+      input:
+        # "Relative path is not allowed to refer to a directory outside its parent"
+        url: ../../skeletons/advanced-workflow-skeleton
+        values:
+          namespace: ${{ parameters.namespace }}
+          argocdNamespace: ${{ parameters.argocdNamespace }}
+          orgName: ${{ parameters.orgName }}
+          repoName: ${{ parameters.repoName }}
+          owner: ${{ parameters.owner }}
+          system: ${{ parameters.system }}
+          description: ${{ parameters.description }}
+          groupId: io.janus.workflow
+          artifactId: ${{ parameters.workflowId }}
+          version: 1.0.0-SNAPSHOT
+          workflowId: ${{ parameters.workflowId }}
+          infrastructureWorkflowId: ${{ parameters.infrastructureWorkflowId }}
+          sourceControl: github.com
+          applicationType: workflow-project
+          lifecycle: development
+          quayOrgName: ${{ parameters.quayOrgName }}
+          quayRepoName: ${{ parameters.quayRepoName }}
+        targetPath: workflow
+    - id: renameFiles
+      action: fs:rename
+      name: Rename files
+      input:
+        files:
+          - from: workflow/src/main/resources/specs/dataindex.yaml
+            to: workflow/src/main/resources/specs/${{ parameters.workflowId }}-dataindex.yaml
+            overwrite: true
+          - from: workflow/src/main/resources/template.sw.yaml
+            to: workflow/src/main/resources/${{ parameters.workflowId }}.sw.yaml
+            overwrite: true
+    - id: buildCodeTemplate
+      name: Generating the Build Code for the Workflow
+      action: fetch:template
+      if: ${{ parameters.CI == 'tekton_argocd' }}
+      input:
+        url: ../../build
+        values:
+          namespace: ${{ parameters.namespace }}
+          argocdNamespace: ${{ parameters.argocdNamespace }}
+          orgName: ${{ parameters.orgName }}
+          workflowId: ${{ parameters.workflowId }}
+          gitUrl: git@github.com:${{ parameters.orgName }}/${{ parameters.repoName }}.git
+          gitOpsUrl: git@github.com:${{ parameters.orgName }}/${{ parameters.repoName }}-gitops.git
+          quayOrgName: ${{ parameters.quayOrgName }}
+          quayRepoName: ${{ parameters.quayRepoName }}
+          gitHost: ${{ parameters.gitlabHost }}
+          convertToFlat: false
+        targetPath: workflow
+    - id: addCIPipeline
+      name: Adding the gitlab pipeline for the Workflow
```

Comment:
generally speaking, we wanted shorten titles for steps (more concise so they fit into the screen)